starknet_py/net/account/account_client.py

```diff
@@ -152,11 +152,11 @@ async def create_account(
         """
         Creates the account using
         `OpenZeppelin Account contract
-        <https://github.com/OpenZeppelin/cairo-contracts/blob/main/contracts/Account.cairo>`_
+        <https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/Account.cairo>`_
```

Comment:
Actually we currently use [this contract](https://github.com/starkware-libs/cairo-lang/blob/4e233516f52477ad158bc81a86ec2760471c1b65/src/starkware/starknet/third_party/open_zeppelin/Account.cairo) 
We should probably also change that in other places

Reply:
Are there any other places? I've searched, but I didn't find it

Reply:
If that's the case just change it here. New account contract is already referenced in 0.8.0 [migration guide](https://starknetpy.readthedocs.io/en/latest/migration_guide.html).